go.mod

```diff
@@ -5,10 +5,10 @@ go 1.20
 require (
```

Comment:
again the joys of go.mod -  over 1000 files changed - roll on go workspaces

Reply:
I know :/
I separated most of those in a specific commit to facilitate readability of the PR.
This makes it a higher risk change. cc @zhouying7780 @kasturinarra